### ISSUE TITLE
Crash on startup - Steamworks SDK crashes

### ISSUE THREAD

**Issue by conifer-dev** [muttleyxd/arma3-unix-launcher#206]
Tried using .DEB package and .AppImage but both return the same issue:

```
[2022-03-06 10:21:53.126] [info] Started Arma 3 Unix Launcher, config file path: "/home/conifer/.config/a3unixlauncher/launcher.conf"
[S_API] SteamAPI_Init(): Loaded '/home/conifer/.steam/debian-installation/linux64/steamclient.so' OK.
Setting breakpad minidump AppID = 107410
SteamInternal_SetMinidumpSteamID:  Caching Steam ID:  [MySteamID - obviously removed it from here.] [API loaded no]
[2022-03-06 10:21:54.214] [warning] exception while parsing settings["protonDisableEsync"]:
[json.exception.out_of_range.403] key 'protonDisableEsync' not found
```

I'm on Debian 11 sid, not running Steam flatpak and using ProtonGE if it matters.

**Comment by muttleyxd** [comment]
This crash probably isn't related to the last message (about protonDisableEsync)

can you post logs with `--verbose` option? `arma3-unix-launcher --verbose`

**Comment by conifer-dev** [comment]
> 

This is what I get back:
```
conifer@conifer:~$ arma3-unix-launcher --verbose
[2022-03-06 11:10:35.099] [info] Started Arma 3 Unix Launcher, config file path: "/home/conifer/.config/a3unixlauncher/launcher.conf"
[2022-03-06 11:10:35.100] [debug] arma path: '/mnt/Games/SteamLibrary/steamapps/common/Arma 3', workshop path: '/mnt/Games/SteamLibrary/steamapps/workshop/content/107410'
[S_API] SteamAPI_Init(): Loaded '/home/conifer/.steam/debian-installation/linux64/steamclient.so' OK.
Setting breakpad minidump AppID = 107410
SteamInternal_SetMinidumpSteamID:  Caching Steam ID:  [MySteamID] [API loaded no]
[2022-03-06 11:10:36.304] [trace] trying to get workshop mods
[2022-03-06 11:10:36.335] [warning] exception while parsing settings["protonDisableEsync"]:
[json.exception.out_of_range.403] key 'protonDisableEsync' not found
Segmentation fault
```

**Comment by muttleyxd** [comment]
Okay, this is just magic.

Three things you can try
- remove `~/.config/a3unixlauncher/launcher.conf` file, maybe it's somehow related to this issue, but I don't think so
- exit Steam before you start the launcher - this will disable Steamworks SDK initialization (which is used for better integration with workshop, but launcher works without it too) - maybe the crash is inside Steamworks
- try building the launcher on your own
```
git clone https://github.com/muttleyxd/arma3-unix-launcher.git
cd arma3-unix-launcher
mkdir build
cd build
cmake ..
make

./src/arma3-unix-launcher/arma3-unix-launcher
```

**Comment by conifer-dev** [comment]
The only workaround it was to build the launcher myself.

**Comment by muttleyxd** [comment]
So it looks like a crash in Steamworks SDK. I think I'll have to keep this issue open - some kind of workaround will be required.

**Comment by DanielGaaA** [comment]
I have same issue. Latest working build is `arma3-unix-launcher commit 334` or building it myself

**Comment by muttleyxd** [comment]
@DanielGaaA @conifer-dev 
This should be fixed by #211. I have updated Steamworks SDK to 1.53a (newest version on this day). If it still crashes for you then please use `--disable-steam-integration` parameter for the launcher.

Fell free to reopen if issue persists
